src/ufo.rs

```diff
@@ -124,6 +126,42 @@ impl Ufo {
         }
     }
 
+    /// Attempt to save this UFO to the given path, overriding any existing contents.
+    ///
+    /// This may fail; instead of saving directly to the target path, it is a good
+    /// idea to save to a temporary location and then move that to the target path
+    /// if the save is successful.
+    pub fn save(&self, path: impl AsRef<Path>) -> Result<(), Error> {
+        let path = path.as_ref();
+        self.save_impl(path)
+    }
+
+    fn save_impl(&self, path: &Path) -> Result<(), Error> {
+        if self.meta.creator.as_str() != DEFAULT_METAINFO_CREATOR {
+            return Err(Error::NotCreatedHere);
+        }
+
+        fs::remove_dir_all(path)?;
```

Comment:
Fails when dir doesn't already exist. Should be `if path.exists(): remove it`.

Reply:
ah yea I bumped into that when trying to use this too, have it patched locally but forgot to update. :)

Reply:
You may want an `overwrite: bool` parameter on the function.